armi/interfaces.py

```diff
@@ -153,6 +153,11 @@ def __init__(self, r, cs):
         self.cs = settings.getMasterCs() if cs is None else cs
         self.r = r
         self.o = r.o if r else None
+        # default interface variables used for tight coupling
+        self.tightCouplingTolerance = 1e-4
```

Comment:
This seems like something that should be configured globally from case settings or by interface. I would suggest that for the general interface that this is _not_ defined and then raises an `Exception` of some sort when called upon but not implemented.

Reply:
It might be reasonable to set this up in the `interactInit` call.

Reply:
Good feedback. I went with the case settings route and added a settings validator. Let me know if you had something else in mind. 
https://github.com/terrapower/armi/pull/1033/commits/931763e5fca8bff014236d1c1a382efe43005589